src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1703,6 +1703,284 @@ int p
                 }
                 """);
 
+            // line breaks between property and other member
+            TestNormalizeDeclaration(
+                "class A{public string Prop{get;}public int f;}", """
+                class A
+                {
+                  public string Prop { get; }
+
+                  public int f;
+                }
+                """);
+            TestNormalizeDeclaration(
+                """class A{public string Prop{get;}="xyz";public int f;}""", """
+                class A
+                {
+                  public string Prop { get; } = "xyz";
+
+                  public int f;
+                }
+                """);
+            TestNormalizeDeclaration(
+                "class A{public string Prop{get;set;}public int f;}", """
+                class A
+                {
+                  public string Prop { get; set; }
+
+                  public int f;
+                }
+                """);
+            TestNormalizeDeclaration(
+                """class A{public string Prop{get;set;}="xyz";public int f;}""", """
+                class A
+                {
+                  public string Prop { get; set; } = "xyz";
+
+                  public int f;
+                }
+                """);
+            TestNormalizeDeclaration(
+                "class A{public string Prop{get;}public int Prop2{get;set;}}", """
+                class A
+                {
+                  public string Prop { get; }
+
+                  public int Prop2 { get; set; }
```

Comment:
this test is the thing i'm the most wary about.  do we normally group fields together without newlines between them?  if so, i think we should default to properties being handled the same way.  i'd prefer we resolve this here so we don't have to have multiple PRs changing this.   I could also be convinced this is fine... but i'd like the thoughts of @333fred @jcouv .  To me, even looking at our codebase, props are aggregated together.  So preserving that would be good.